filechanges/opt/musicbox/startup.sh

```diff
@@ -224,13 +224,26 @@ fi
 # renice mopidy to 19, to have less stutter when playing tracks from spotify (at the start of a track)
 renice 19 `pgrep mopidy`
 
-if [ "$INI__musicbox__autoplay" -a "$INI__musicbox__autoplaywait" ]
+if [ "$INI__musicbox__autoplay" -a "$INI__musicbox__autoplaymaxwait" ]
```

Comment:
The script now checks to make sure that the value specified for ```autoplaymaxwait``` is indeed a number. Defaults to 60 if not.